nservicebus/msmq/scaling-out_overview_core_6.partial.md

```diff
@@ -16,4 +16,6 @@ Following table compares both approaches:
 | Round-robin load balancing                  | Fair load balancing                         |
 
 
-WARNING: When using sender-side distribution scaling out technique in a mixed version environment make sure to deploy a distributor in front of the scaled out version 6 endpoint if that endpoint needs to communicate with endpoints using versions lower than 6 (refer to [the distributor sample](/samples/scaleout/distributor/) for details). Otherwise event messages will get duplicated and commands won't be properly distributed.
\ No newline at end of file
+WARNING: When using sender-side distribution scaling out technique in a mixed version environment make sure to deploy a distributor in front of the scaled out version 6 endpoint if that endpoint needs to communicate with endpoints using versions lower than 6 (refer to [the distributor sample](/samples/scaleout/distributor/) for details). Otherwise event messages will get duplicated and commands won't be properly distributed.
```

Comment:
That's quite a long and complicated sentence. I think it could at least use a comma after `[...] in a mixed version environment`